qiita_db/analysis.py

```diff
@@ -1011,6 +1011,10 @@ def _build_mapping_file(self, samples, rename_dup_samples=False):
                     samps = set(samps) - all_ids
                     all_ids.update(samps)
 
+                # appending the study title
+                qm['qiita_study_title'] = qdb.artifact.Artifact(
```

Comment:
Correct. Basically, this way we make sure that legacy and new ones have it. Also the reason I prepended with qiita_